src/sentry/utils/jwt.py

```diff
@@ -49,7 +50,7 @@ def decode(
     *,  # Force passing optional arguments by keyword
     audience: Optional[Union[str, bool]] = None,
     algorithms: Optional[List[str]] = None,
-) -> JSONData:
+) -> dict[str, JSONData]:
```

Comment:
it was mad about "returning Any" since `JSONData` is an alias to `Any` -- I just pushed the `Any` one level deeper (and matched the annotation from `jwt`)